upup/pkg/kutil/drain.go

```diff
@@ -0,0 +1,674 @@
+/*
+Copyright 2016 The Kubernetes Authors.
```

Comment:
Why are we copy/pasting code from the main repo into kops? Can we just reference the library directly so we only have one source of truth? This seems a bit worrisome that we are attempting to import an almost verbatim copy of [drain.go](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/drain.go) into kops.. Whats the advantage? And why `kutil`?